enclave-tls/samples/enclave-tls-client/client.c

```diff
@@ -95,6 +95,7 @@ int enclave_tls_echo(int fd, enclave_tls_log_level_t log_level,
 		len = sizeof(buf) - 1;
 	buf[len] = '\0';
 
+#ifdef SGX
```

Comment:
There is no need to add `#ifdef SGX`. The `len` is decided by the server.

Reply:
Nop. sgx_measurement_t is undefined in normal build. So maybe we have a gap about the meaning of SGX. In my opinion, SGX means "yes I have installed sgx sdk environment and intend to build enclave-tis working for sgx enclave". If SGX is not specified, it is unpredictable whether referring to sgx related definition is applicable.

Reply:
Then please add OCCLM macro, because we need to print `mencalve` and `mesigner` when client running in occlum environment.